webapp/src/Utils/Utils.php

```diff
@@ -865,6 +874,9 @@ public static function parseTsvLine(string $line): array
 
     /**
      * Reindex the given array by applying the callback to each item.
+     *
+     * @param array<int|string, mixed> $array
+     * @return array<int|string, mixed>
```

Comment:
This could become something like:
```suggestion
	 * @template TKey of array-key
     * @template UKey of array-key
     * @template V
     * @param array<TKey, V> $array
     * @param callable(V, TKey): UKey $callback
     * @return array<UKey, V>
```